download/Cargo.toml

```diff
@@ -23,6 +23,8 @@ thiserror.workspace = true
 url.workspace = true
 
 [dev-dependencies]
-hyper = { version = "0.14", default-features = false, features = ["tcp", "server"] }
+hyper = { version = "1.0", default-features = false, features = ["server", "http1"] }
+hyper-util = { version = "0.1.1", features = ["tokio"] }
+http-body-util = "0.1.0"
```

Comment:
You might want to sort the deps alphabetically here.

Reply:
Done. Nice habit :D